GOVERNANCE.md

```diff
@@ -0,0 +1,182 @@
+particularly with respect to its strategic direction and long-term health, may
+be nominated to become a maintainer. This role is described below.
+
+## Maintainers 
+
+Maintainers are community members who have shown that they are committed to the
+continued development of the project through ongoing engagement with the
+community.
+
+Typically, a potential maintainer will need to show that they have an
+understanding of the project, its objectives and its strategy. They will also
+have provided valuable contributions to the project over a period of time.
+
+The project maintainers consists of those individuals identified as
+‘project owners’ on the development site. The maintainers have additional
+responsibilities over and above those of a contributor. These responsibilities
+ensure the smooth running of the project. Maintainers are expected to review
+and merge code contributions, participate in strategic planning, approve
+changes to the governance model and manage the copyrights within the project
+outputs.
+
+Maintainers do not have significant authority over other members of the
+community, although they make decisions when community consensus cannot be
+reached. In addition, maintainers may interact through the project’s private
+mailing list if it exists or directly based on the maintainers list.
+This list is used for sensitive issues, such as votes for new maintainers and
+legal matters that cannot be discussed in public. It is never used for project
+management or planning.
+
+Maintainer membership is by invitation from the existing members. A nomination
+will result in discussion and then a vote by the existing maintainers on the
+project’s private list.
+Maintainer membership votes are subject to consensus approval of the current
+maintainer members.
+
+# Support
+
+All participants in the community are encouraged to provide support for new
+users within the project management infrastructure. This support is provided as
+a way of growing the community. Those seeking support should recognise that all
+support activity within the project is voluntary and is therefore provided as
+and when time allows. A user requiring guaranteed response times or results
+should therefore seek to purchase a support contract from a community member.
+However, for those willing to engage with the project on its own terms, and
+willing to help support other users, the community support channels are ideal.
+
+# Contribution process
+
+Anyone can contribute to the project, regardless of their skills, as there are
+many ways to contribute. For instance, a contributor might be active on the
+project mailing list and issue tracker, or might supply patches.
+
+# Decision making process
+
+Decisions about the future of the project are made through discussion with all
+members of the community, from the newest user to the most experienced
+maintainer member. All non-sensitive project management discussion takes place
```

Comment:
> All non-sensitive project management discussion takes place on the project developer mailing list.
This is not true for Nmstate because most project management discussion takes place on internal IRC, in video conferences followed by PRs.

Reply:
On IRC or video conferences we are taking the decisions in a non formal manner.
If we want to make something formal, we need an email.

At the moment, we are good with this informal way of doing things and I am fine with it.
We did use the email for some issues, some even prefer it over the informal way.

Reply:
I do not follow. You like using IRC/video conferences but at the same time you propose a document that says the exact opposite because it says *all* of this discussion takes place on the mailing list. Also I do not agree that IRC/video conferences cannot be formal. This document describe the way things are happening. So just writing down that all discussion happens in closed meetings makes it the formal process.

Reply:
I agree with the text: in case one wants to formalize a discussion, it needs to be done through mailing lists.

But as you are that passionate about this, I will add "irc" and "video conferences".